### ISSUE TITLE
Getting error please help

### ISSUE THREAD

**Issue by froedevf** [catslover32/Nightscout-for-Alexa#1]
Getting "The remote endpoint could not be called, or the response it returned was invalid." when using the Service Simulator.

Btw really appreciate the effort you put in here, excited to get this working!

Apologies on not following the directions and PMing, couldn't find out how to do that.

Thanks,
-Dave

**Comment by froedevf** [comment]
Got a little further following google advice on resolving the error, so it shows me:
START RequestId: 0c724e29-b20f-11e6-85e4-f36b1e4a56c3 Version: $LATEST
Unable to import module 'lambda_function': No module named requests

END RequestId: 0c724e29-b20f-11e6-85e4-f36b1e4a56c3
REPORT RequestId: 0c724e29-b20f-11e6-85e4-f36b1e4a56c3	Duration: 0.39 ms	Billed Duration: 100 ms 	Memory Size: 128 MB	Max Memory Used: 8 MB

So maybe I'm creating the zip wrong or uploading wrong, tried a few times but no luck.

Thanks again,
-Dave

**Comment by JonathanM32** [comment]
Yeah, triple check that your zip file has both of the folders with requests
in it.

**Comment by JonathanM32** [comment]
I think it might work now. I just made some changes to the code. Try re uploading and tell me if it works

**Comment by froedevf** [comment]
Unfortunately same error.

**Comment by froedevf** [comment]
Tried creating the zip on a mac as well, same issue, sorry must be repeatedly doing something stupid.
